test/simplifyExpression/functionsSearch/absoluteValue.test.js

```diff
@@ -10,6 +10,9 @@ describe('abs', function () {
   const tests = [
     ['abs(4)', '4'],
     ['abs(-5)', '5'],
+    ['abs(-x)', 'x'],
+    ['abs(-xy)', 'xy'],
```

Comment:
(that's why we had the check in unsupporteNodes that the argument to abs was a constant, cause otherwise it gets more complicated)

technically abs(-x) is

- x when x >= 0
- -x when x < 0